ci-operator/config/openshift/open-telemetry-opentelemetry-operator/openshift-open-telemetry-opentelemetry-operator-main__ocp-4.14-amd64.yaml

```diff
@@ -76,9 +78,9 @@ tests:
       BASE_DOMAIN: qe.devcluster.openshift.com
       EO_SUB_CHANNEL: stable-5.7
       EO_SUB_SOURCE: redhat-operators
-      SKIP_TESTS: tests/e2e/instrumentation-sdk tests/e2e/instrumentation-go tests/e2e/instrumentation-apache-multicontainer
-        tests/e2e/instrumentation-apache-httpd tests/e2e/targetallocator-features
-        tests/e2e/prometheus-config-validation tests/e2e/smoke-targetallocator
+      SKIP_TESTS: tests/e2e/targetallocator-features tests/e2e/prometheus-config-validation
+        tests/e2e/smoke-targetallocator tests/e2e-instrumentation/instrumentation-apache-httpd
+        tests/e2e-openshift/otlp-metrics-traces
```

Comment:
Why do we skip this test?

Reply:
The otlp metrics one I have fixed upstream but our forked repos need to sync for us to enable it. The rest are failing, I’ll debug them next week.